tokenserver/assignment/memorynode.py

```diff
@@ -19,6 +20,7 @@ def __init__(self, service_entry=None, **kw):
         self._service_entry = service_entry
         self._users = {}
         self._next_uid = 1
+        self.settings = get_current_registry().settings
```

Comment:
Actually, it's not clear to me why this needs to access the settings differently than the SQL backend. I think it should be able to obtain settings via `**kw` argument in the same way that the SQL backend does, since the backend should be loaded via abstract interface [here](https://github.com/mozilla-services/tokenserver/blob/daf36fb991e0b3fe34dd21d62c20d192f968ff44/tokenserver/__init__.py#L40)

Perhaps something in the tests is initializing the `MemorynodeAssignmentBackend` differently? If so, I think we should find and fix it there rather than handle settings differently between the two backends.

Reply:
hrm. I think part of the problem is that `get_current_registry()` is a Pyramid function that returns the settings defined within the Pyramid framework. Each setting is namespaced. 

The `kw` argument, however, is a local calling parameter, so the settings aren't namespaced. 

This is a bit more complicated because the nose tests don't always specify either the kw or the pyramid registry. I'll try to normalize them, but might be a bit more than the scope of this PR.